libs/react-textarea-autogrow/src/index.js

```diff
@@ -0,0 +1,3 @@
+import AutogrowTextarea from './lib/react-textarea-autogrow';
+
```

Comment:
@Sowmyareddy22 Without directly exporting default Autogrow Text Area, can we alsoe export the useAutogrow hook too, in case users want to implement directly using the hook.

Makes sense?

Reply:
@rjvim Yes possible.

Reply:
@Sowmyareddy22 Let's do that, and we can do like export:

`export {AutogrowTextarea, useAutoGrow};`